powerapps-docs/developer/data-platform/fetchxml/join-tables.md

```diff
@@ -99,7 +99,7 @@ The previous example is a many-to-one relationship where many account records ca
 
 ### Retrieve relationship information
 
-If you use the [XrmToolbox](../community-tools.md#xrmtoolbox) [FetchXmlBuilder](https://fetchxmlbuilder.com/), you can see how this tool allows you to select the relationship to set the appropriate `name`, `from`, and `to` attribute values.
+If you use the [XrmToolbox](../community-tools.md#xrmtoolbox) [FetchXML Builder](https://fetchxmlbuilder.com/), you can see how this tool allows you to select the relationship to set the appropriate `name`, `from`, and `to` attribute values.
```

Comment:
@rappen @JimDaly  what about the `XrmToolBox` name that is written as `XrmToolbox` 😛

Reply:
How how HOW could I miss that... 😣